WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -485,6 +506,15 @@ extension AztecPostViewController {
     enum EditionMode {
         case richText
         case html
+
+        mutating func toggle() {
```

Comment:
@SergioEstevao Media is, as of now, unimplemented in Aztec's WPiOS Integration, and an assertion is actually expected. 

As per Links > Crashes, cannot repro here. I'll file another issue to track that, thanks!